docs/topics/sb-features-and-frameworks-summary.adoc

```diff
@@ -75,7 +75,7 @@ a|
 No
 
 a|
-Spring Cloud Kubernetes - Ribbon
+link:https://github.com/spring-cloud-incubator/spring-cloud-kubernetes[Spring Cloud Kubernetes] - Ribbon
```

Comment:
Ribbon is actually also Netflix, right @metacosm?

Reply:
OK, wrong, there's a Ribbon module in Spring Cloud Kubernetes. Not that I would understand it :-)

Reply:
I guess both would be correct, here… We're not using either in the boosters at the moment, though.